api.raml

```diff
@@ -90,7 +91,7 @@ types:
         "active": "yes",
         "locked": "no",
         "last_login_utc": "2017-05-24 14:04:51 UTC",
-        "do_not_disclose": false,
+        "do_not_disclose": "no",
```

Comment:
:-D  oh man, this double-negative is a little hard to read...it might've been better once we realized that yes/no stuff to name this `disclose` or `sensitive`... not a biggie I don't suppose at this point though.

Reply:
I think `disclose` is better but let me check the thesaurus and see if there's something better.

Reply:
unveil :-D

Reply:
is *protect* any better than disclose?

Reply:
I would prefer a word with positive logic (antonym of disclose) so that the true/false sense doesn't change. In other words, I'd still like to have "no" be the default.

Reply:
How about _conceal_?

Reply:
oh yeah, I like that.

Reply:
sure, that works for me

Reply:
It looks like you've come to a consensus already, so feel free to reject this, but would "hide" be any better than "conceal"? It just seems like a word we'd be more likely to use in a normal conversation.